### DISCUSSION TITLE
Task lists clobber bullet point items

### DISCUSSION THREAD

**Discussion by choldgraf** [community/community#44107] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

If you put a bullet point list inside of a task list block, then it will remove all of the list items! This is probably somewhat common if people forget to add the `[ ]` to items at the first pass.

Instead, I think you should either (1) retain the bullet list and make the tasklist a `literal` block until it has the right syntax. or (2) convert bullet point items into empty task items.

For example here's a little GIF of this behavior:

![chrome_rK1AB78Puh](https://user-images.githubusercontent.com/1839645/212077330-a5f2a593-30ce-40f1-85dc-638f324dc9be.gif)

**Comment by abernix** [comment]
A similar clobbering will occur if you put a block of text under the checklist.  For example:


#### Tasklist markdown checkboxs: `Some text under it` will be clobbered
``````
```[tasklist]
- [ ] This item
      Some text under it
```
``````

I find this to be a surprising behaviour since the existing checklist markdown approach would allow this to be text that is _part_ of the checkbox... e.g., 

#### Traditional markdown checkboxs: `Some text under it` will NOT be clobbered.

``````
```
- [ ] This item
      Some text under it
```
``````

Saving the first will clobber the "Some text under it" whereas the second will
